use/LMM-Genetic-Diversity.Rmd

```diff
@@ -0,0 +1,152 @@
+
+## Packages
+
+Packages required:
+
+Loading the required packages:  
+
+```{r, packages, message=FALSE}
+library("lme4")
+library("MuMIn")
+library("multcomp")
+```
+
+# Analysis
+
+## Section 1: Load the data
+
+At first we need to load our dataset. We will import "GenDivEuglossa.txt". 
+
+```{r, load_data_show, eval=FALSE}
+GenDivEuglossa <- read.delim("GenDivEuglossa.txt", header = TRUE)
+summary(GenDivEuglossa)
+str(GenDivEuglossa)
+```
+
+```{r, load_data_evaluate, echo=FALSE}
+# Note: the code chunk above will be shown to the reader, but it will not run.
+# This code chunk will run, but will not be shown to the reader.
+GenDivEuglossa <- read.delim("GenDivEuglossa.txt", header = TRUE)
+summary(GenDivEuglossa)
+str(GenDivEuglossa)
+```
+
+
+## Section 2: Data analysis/Checking assumptions  
+
+Second, we build our model using the function lmer (). With this command, we create a model with Allellic Richness as response with Management and Locus as fixed and random effects factors respectively.
+
+```{r, load_data_show, eval=FALSE}
+modelMANAG<-lmer(AllelicRichness~Management+(1|Locus), GenDivEuglossa)
+summary
+```
+
+```{r, load_data_evaluate, echo=FALSE}
+# Note: the code chunk above will be shown to the reader, but it will not run.
+# This code chunk will run, but will not be shown to the reader.
+modelMANAG<-lmer(AllelicRichness~Management+(1|Locus), GenDivEuglossa)
+summary(modelMANAG)
+```
+
+Now, we check our model assumptions. We check for normality and homogeneity by inspecting the residual plot. This plot depicts fitted values on the x-axis and residuals on the y-axis.
+
+```{r, load_data_show, eval=FALSE}
+plot(modelMANAG)
+```
+
+```{r, load_data_evaluate, echo=FALSE}
+# Note: the code chunk above will be shown to the reader, but it will not run.
+# This code chunk will run, but will not be shown to the reader.
+plot(modelMANAG)
+```
+
+## Section 3: Summary statistics 
+
+In order to see the effect of our fixed factor 'Management', and compare the performance of our mixed model, we run a likelihood ratio test, comparing our model (modelMANAG), which includes the fixed factor "Management", with a model (modelNULL) that excludes it.
+
+```{r, load_data_show, eval=FALSE}
+modelMANAG<-lmer(AllelicRichness~Management+(1|Locus), GenDivEuglossa, REML=FALSE)
+modelNULL<-lmer(AllelicRichness~1+(1|Locus), GenDivEuglossa, REML=FALSE)
+anova(modelMANAG, modelNULL)
+```
+
+```{r, load_data_evaluate, echo=FALSE}
```

Comment:
This chunk is not necessary. You can delete it and remove `eval = FALSE` from the chunk above